ksqldb-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -447,20 +447,26 @@ public void shouldToCallOutToKafkaIfDeleteListIsEmpty() {
   public void shouldDeleteInternalTopics() {
     // Given:
     final String applicationId = "whatEva";
-    final String internalTopic1 = applicationId + "-something-repartition";
-    final String internalTopic2 = applicationId + "-someting-changelog";
+    final String internalTopic1 = applicationId
+        + "-KTABLE-FK-JOIN-SUBSCRIPTION-REGISTRATION-0000000012-topic";
+    final String internalTopic2 = applicationId
+        + "-KTABLE-FK-JOIN-SUBSCRIPTION-RESPONSE-0000000012-topic";
+    final String internalTopic3 = applicationId + "-something-repartition";
+    final String internalTopic4 = applicationId + "-someting-changelog";
```

Comment:
nit: typo in something

Reply:
Fixed in https://github.com/confluentinc/ksql/pull/8520/commits/0c9865e57da8e53df2ca3191011ffaa8d5bf5be6#diff-dfb8a82a63edbd6892fa746e000c798422084f930fb0499e1d434f9bcf32ac9cR454-R455.